sdk/search/search-documents/test/public/node/searchClient.spec.ts

```diff
@@ -17,40 +17,37 @@ import {
   AzureKeyCredential,
 } from "../../../src";
 import { Hotel } from "../utils/interfaces";
-import { createIndex, populateIndex, WAIT_TIME, createRandomIndexName } from "../utils/setup";
+import { createIndex, populateIndex, WAIT_TIME } from "../utils/setup";
 import { delay, serviceVersions } from "../../../src/serviceUtils";
```

Comment:
This too @HarshaNalluru

Reply:
Not doing it here to keep the changes minimal